microbench/triangle-counting/src/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2023. University of Texas at Austin. All rights reserved.
+
+#
+# pando-triangle-counting target
+#
+
+target_include_directories(pando-triangle-counting
+    PUBLIC
+        "$<BUILD_INTERFACE:${pando-tc-galois_SOURCE_DIR}/include>"
+        "$<BUILD_INTERFACE:${pando-lib-galois_SOURCE_DIR}/include>"
+        "$<BUILD_INTERFACE:${pando-lib-galois_BINARY_DIR}/include/pando-lib-galois>"
+        ${pando-rt_BINARY_DIR}/include
+)
+
+target_sources(pando-triangle-counting
+    PRIVATE
+        ${CMAKE_CURRENT_LIST_DIR}/utils.cpp
+        #${CMAKE_CURRENT_LIST_DIR}/tc_algos.cpp
+)
+
+target_link_libraries(pando-triangle-counting
+    PUBLIC pando-lib-galois::pando-lib-galois)
+
+#if (PANDO_RT_BACKEND STREQUAL "DRVX")
+#    target_link_libraries(pando-triangle-counting PUBLIC "$<LINK_LIBRARY:WHOLE_ARCHIVE,pando-rt::pando-rt>")
+#else()
+#    target_link_libraries(pando-triangle-counting
+#    PUBLIC
+#        pando-rt::pando-rt
+#)
+#endif()
```

Comment:
Please delete the unused comments.

Reply:
Done in https://github.com/utcs-scea/pando-mono/pull/45/commits/73e0fe695a28426ffb33fd0fa32a46c59ffb039c